agents/terminal/src/main/resources/installers/1.0.1/org.eclipse.che.terminal.json

```diff
@@ -0,0 +1,15 @@
+{
+  "id": "org.eclipse.che.terminal",
+  "version": "1.0.0",
+  "name": "Terminal",
+  "description": "Embedded web terminal",
+  "dependencies": [],
+  "properties": {},
+  "servers": {
+    "terminal": {
+      "port": "4411/tcp",
```

Comment:
Does it make sense to declare port if it might be reassigned.

Reply:
Yes. It is default port. It will be reassigned only if it conflicts with another installer.